admin.go

```diff
@@ -55,6 +56,34 @@ func (om *OpenMock) StartAdmin() {
 		return c.String(200, string(b))
 	})
 
+	e.DELETE("/api/v1/templates/:key", func(c echo.Context) error {
+		key, err := url.QueryUnescape(c.Param("key"))
+		if err != nil {
+			return c.String(400, fmt.Sprintf("Invalid Key: %v. error: %s", key, err))
+		}
+
+		mock := om.repo.FindByKey(key)
```

Comment:
nit, do you need this? I think you can just try HGET in redis, since the delete only applies to redis storage.

Reply:
Updated to check redis and return the deleted yaml if its successfully deleted